diff --git a/Python27/Lib/distutils/msvccompiler.py b/Python27/Lib/distutils/msvccompiler.py
index d5cd403..8695dc6 100644
--- a/Python27/Lib/distutils/msvccompiler.py
+++ b/Python27/Lib/distutils/msvccompiler.py
@@ -156,7 +156,7 @@ def get_build_version():
     For Python 2.3 and up, the version number is included in
     sys.version.  For earlier versions, assume the compiler is MSVC 6.
     """
-
+    return 10.0
     prefix = "MSC v."
     i = string.find(sys.version, prefix)
     if i == -1:
@@ -654,6 +654,6 @@ class MSVCCompiler (CCompiler) :
 if get_build_version() >= 8.0:
     log.debug("Importing new compiler from distutils.msvc9compiler")
     OldMSVCCompiler = MSVCCompiler
-    from distutils.msvc9compiler import MSVCCompiler
+    from distutils.msvc10compiler import MSVCCompiler
     # get_build_architecture not really relevant now we support cross-compile
-    from distutils.msvc9compiler import MacroExpander
+    from distutils.msvc10compiler import MacroExpander
