--- uuid.py.orig	Tue Apr 10 00:07:32 2012
+++ uuid.py	Tue Oct 22 14:58:10 2013
@@ -397,17 +397,18 @@
 try:
     import ctypes, ctypes.util
 
-    # The uuid_generate_* routines are provided by libuuid on at least
-    # Linux and FreeBSD, and provided by libc on Mac OS X.
-    for libname in ['uuid', 'c']:
-        try:
-            lib = ctypes.CDLL(ctypes.util.find_library(libname))
-        except:
-            continue
-        if hasattr(lib, 'uuid_generate_random'):
-            _uuid_generate_random = lib.uuid_generate_random
-        if hasattr(lib, 'uuid_generate_time'):
-            _uuid_generate_time = lib.uuid_generate_time
+    if os.name not in ['nt', 'ce']:
+        # The uuid_generate_* routines are provided by libuuid on at least
+        # Linux and FreeBSD, and provided by libc on Mac OS X.
+        for libname in ['uuid', 'c']:
+            try:
+                lib = ctypes.CDLL(ctypes.util.find_library(libname))
+            except:
+                continue
+            if hasattr(lib, 'uuid_generate_random'):
+                _uuid_generate_random = lib.uuid_generate_random
+            if hasattr(lib, 'uuid_generate_time'):
+                _uuid_generate_time = lib.uuid_generate_time
 
     # The uuid_generate_* functions are broken on MacOS X 10.5, as noted
     # in issue #8621 the function generates the same sequence of values
